components/brave_rewards/browser/publisher_info_database.cc

```diff
@@ -1044,29 +1044,100 @@ bool PublisherInfoDatabase::MigrateV1toV2() {
     return false;
   }
 
-  if (!CreateContributionInfoTable()) {
+  const char* name = "contribution_info";
+  sql = "CREATE TABLE ";
+  sql.append(name);
+  sql.append(
+      "("
+      "publisher_id LONGVARCHAR,"
+      "probi TEXT \"0\"  NOT NULL,"
+      "date INTEGER NOT NULL,"
+      "category INTEGER NOT NULL,"
+      "month INTEGER NOT NULL,"
+      "year INTEGER NOT NULL,"
+      "CONSTRAINT fk_contribution_info_publisher_id"
+      "    FOREIGN KEY (publisher_id)"
+      "    REFERENCES publisher_info (publisher_id)"
+      "    ON DELETE CASCADE)");
+  if (!GetDB().Execute(sql.c_str())) {
+    return false;
+  }
+
+  if (!GetDB().Execute(
+      "CREATE INDEX IF NOT EXISTS contribution_info_publisher_id_index "
+      "ON contribution_info (publisher_id)")) {
     return false;
   }
 
-  if (!CreateContributionInfoIndex()) {
+  // Recurring_donation
+  const char* recurring_donation = "recurring_donation";
```

Comment:
could this just be `name = ...` like in other places?

Reply:
Done.